tests/detect-bytemath-07/test.rules

```diff
@@ -0,0 +1,5 @@
+alert tcp any any -> any any (msg:"byte_math varname test sig"; \
```

Comment:
Readme linking to issue would be nice

We want to test somehow that the rule currently matches